code/modules/power/supermatter/supermatter_delamination/_sm_delam.dm

```diff
@@ -9,6 +9,7 @@ GLOBAL_LIST_INIT(sm_delam_list, list(
 /// Logic holder for supermatter delaminations, goes off the strategy design pattern.
 /// Selected by [/obj/machinery/power/supermatter_crystal/proc/set_delam]
 /datum/sm_delam
+	COOLDOWN_DECLARE(common_radio_cooldown)
```

Comment:
These datums are used globally and shouldnt hold state. I recommend moving the cooldown to the supermatter itself and have the procs in this datum check that.

Reply:
Sure